tests/core/test_indexset.py

```diff
@@ -48,26 +47,25 @@ def test_create_indexset(self, test_mp, request):
         with pytest.raises(IndexSet.NotUnique):
             _ = run.optimization.indexsets.create("Indexset 1")
 
-    def test_get_indexset(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.runs.create("Model", "Scenario")
+    def test_get_indexset(self, platform: ixmp4.Platform):
+        run = platform.runs.create("Model", "Scenario")
         run.set_as_default()
-        create_indexsets_for_run(platform=test_mp, run_id=run.id, amount=1)
+        create_indexsets_for_run(platform=platform, run_id=run.id, amount=1)
         indexset = run.optimization.indexsets.get("Indexset 1")
         assert indexset.id == 1
         assert indexset.name == "Indexset 1"
 
         with pytest.raises(IndexSet.NotFound):
             _ = run.optimization.indexsets.get("Foo")
 
-    def test_add_elements(self, test_mp, request):
-        test_mp: Platform = request.getfixturevalue(test_mp)  # type: ignore
-        run = test_mp.runs.create("Model", "Scenario")
+    def test_add_elements(self, platform: ixmp4.Platform):
+        run = platform.runs.create("Model", "Scenario")
         test_elements = ["foo", "bar"]
         indexset_1 = run.optimization.indexsets.create("Indexset 1")
-        indexset_1.add(test_elements)
-        run.optimization.indexsets.create("Indexset 2").add(test_elements)
+        indexset_1.add(test_elements)  # type: ignore
+        run.optimization.indexsets.create("Indexset 2").add(test_elements)  # type: ignore
```

Comment:
Why do we need these markers for mypy?

Reply:
I didnt take the time to go through the newly merged changes and fix all the typing issues (i did for ALL the other tests)

Reply:
Unfortunately I don't see mypy's complaints in the GHA runs, so I'll have to remove the markers and resolve the complaints later on.